src/python/bot/minimizer/html_minimizer.py

```diff
@@ -16,6 +16,7 @@
 from __future__ import absolute_import
 
 from builtins import object
+from builtins import range
```

Comment:
why is this needed?

Reply:
The linter said that I needed it.

Reply:
More specifically this would be needed to ensure we're using range behavior compatible with Python 3's.